patchwork-gui/src/main/java/net/minecraftforge/client/event/GuiOpenEvent.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2020, 2019-2020
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.client.event;
+
+import net.minecraftforge.eventbus.api.Event;
+
+import net.minecraft.client.gui.screen.Screen;
+
+/**
```

Comment:
Pretty sure this is just old MCP but please change the "gui" names to "screen" where they won't change public method signatures

Reply:
fixed

Reply:
This isn't fixed in the latest commit.

Reply:
should be fixed now
May be last time I forgot to push commits

Reply:
https://github.com/rikka0w0/patchwork-api/blob/feature/GuiOpenEvent/patchwork-gui/src/main/java/net/minecraftforge/client/event/GuiOpenEvent.java

Reply:
In case there's confusion I want the javadoc changed as well

Reply:
fixed as well, in the latest commit